srcpkgs/minijinja-cli/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'minijinja-cli'
+pkgname=minijinja-cli
+version=1.0.12
+revision=1
+build_wrksrc=minijinja-cli
```

Comment:
I tested out that we can instead use

```
configure_args="--workspace --exclude minijinja-py"
make_install_args="--path minijinja-cli"
```

to build/test the whole workspace except for the unused python bindings that require `python3-devel`, the only caveat being we then also need to reset `configure_args` in `post_install`.

Not sure if that would be preferable, but still thought I'd put it out here.

Reply:
What is the benefit of that? That it's running more tests?

Reply:
That comes to mind immediately. I decided to test because using build_wrksrc in this package felt strange, but ultimately if it works maybe no need to overengineer it :P

Reply:
It does run more tests then, but as we've already had with other situations, the `cargo-insta` tests are failing...

Reply:
fair point